app/controllers/account_activations_controller.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
```

Comment:
Why do we need the frozen literals?

Reply:
Based on its configuration, Rubocop currently forces us to include it at the top of every file. 

A good explanation why can be found here: https://stackoverflow.com/questions/37799296/what-does-the-comment-frozen-string-literal-true-do